### ISSUE TITLE
Support \tcolorbox?

### ISSUE THREAD

**Issue by alexlenail** [KaTeX/KaTeX#3091]
For equations in colored boxes (and colored boxes in $$equations$$)

**Comment by edemaine** [comment]
I don't think [tcolorbox](https://tools.ietf.org/doc/texlive-doc/latex/tcolorbox/tcolorbox.pdf) works in math mode, which is the primary goal of KaTeX.  Or are you referring to a different package?

**Comment by alexlenail** [comment]
My mistake! Looking into this some more, getting a colored box around part of an $$equation$$ seems quite involved, and probably out of scope for katex. That's a shame.

**Comment by mbourne** [comment]
Something like this?

![image](https://user-images.githubusercontent.com/7541940/123020502-ea40b900-d404-11eb-9d29-eedf86cf29cf.png)

It involves post-processing.

(1) The magenta border on the numerator of the fraction is rather ugly - I tried to style it better but it's not surprising it failed given the space constraints of the square root.

This was a matter of identifying the element and adding a class to that element.

(2) The green border around the "1 +" involved adding a delimiter to the equation (I used [[ ]] double brackets) to identify the bit I wanted a border around. I then scraped the elements between those delimiters, put them in a new span with the required class, and deleted the original spans.

The second approach may be feasible for you, depending on the complexity of the expression.

Here's the fiddle: https://jsfiddle.net/mbourne/pq3awgfe/

Regards
Murray
